add depencies (#78)
iotaledger/IOTA-2.0-Research-Specifications: 4.7 Markers.md

Note, that we *shall* require that markers are assigned when booking a message. Thus, for that part of the message DAG that is already booked the corresponding marker DAG does not change anymore. 

## 4.7.2 Dependencies

> The *Markers* tool depends on:

Reviewer: ```suggestion
The *Markers* tool specification depends on:
```

> + Tangle: Certain messages in the Tangle are given additional meaning in order to group subsets of messages.

Reviewer: ```suggestion
+ [4.1 - The Tangle](./4.1%20The%20Tangle.md).
```

> + Ledger State: New marker sequences are created when a new conflict is detected.

Reviewer: ```suggestion
+ [5.2 - Ledger State](./5.2%20Ledger%20State.md).
```

Second, assume the existing transaction is not a marker. Then all messages between the transaction and the following future markers (including the transaction itself) get mapped individually to the new branch mapping using the field `IndividualBranchID`. From the future markers onwards, the same applies as in the first scenario. 

For an example implementation of these scenarios also visit the example [here](https://github.com/iotaledger/goshimmer/blob/develop/packages/tangle/images/TestBookerMarkerMappings.md).
## 4.7.7 Marker Application Description

Figure 1 shows an example of how the markers and marker-sequences (here also called Sequence) would look in the Tangle from the perspective of the Message DAG, the marker DAG and the marker-sequence DAG. The purple colored messages are markers:

![](https://i.imgur.com/GENej3O.png)

> **Figure 1: Markers and marker-sequences in the Tangle**

Reviewer: ```suggestion
**Image 4.7.1:** Markers and marker-sequences in the Tangle.
```
